Makefile.include

```diff
@@ -466,6 +466,14 @@ define check_cmd
 	    exit 1;}
 endef
 
+# Check setsid command is available
+#   This is not the case on MacOSX, so it must be built on the fly
+ifeq (, $(shell which setsid))
```

Comment:
The `shell which` patterns are a bit different, check the other ones. I think some system provides an error message on error so cannot rely on the output.
```suggestion
ifeq (0,$(shell which setsid 2>&1 > /dev/null ; echo $$?))
```

Reply:
Thanks, but in fact the right line is:
```mk
ifneq (0,$(shell which setsid 2>&1 > /dev/null ; echo $$?))
```